src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows.xml

```diff
@@ -78,11 +85,15 @@
                 <framework:from>closed</framework:from>
                 <framework:to>review</framework:to>
             </framework:transition>
+            <framework:metadata>
+                <framework:title>workflow title</framework:title>
+            </framework:metadata>
         </framework:workflow>
 
         <framework:workflow name="service_marking_store_workflow" type="workflow">
             <framework:marking-store service="workflow_service"/>
             <framework:support>Symfony\Bundle\FrameworkBundle\Tests\DependencyInjection\FrameworkExtensionTest</framework:support>
+            <!-- This is not using the same format, because we want a test for the old format (BC promise) -->
```

Comment:
`for the other format` as it's not "old" anymore.
And the BC promise parenthesis can probably be removed.